### DISCUSSION TITLE
Job Backup

### DISCUSSION THREAD

**Discussion by fischer70** [jhuckaby/Cronicle#743] [Q&A] [answered]
I've been using Cronicle for a while and very happy with it.  I was wondering if there is a reasonable way to backup the jobs a their schedules.  The logs, job histories and all of that could be lost and I wouldn't care much, but the jobs and their schedules would be a hassle to reassemble.  Any reasonable way to do this?  Thanks!

**Comment by jhuckaby** [comment, marked as answer]
This will backup everything except job logs and job history: https://github.com/jhuckaby/Cronicle/blob/master/docs/CommandLine.md#data-import-and-export

**Reply by fischer70** [reply]
Thank you!  Exactly what I want and embarrassed I didn't see that in the docs!

**Reply by jhuckaby** [reply]
All good -- it's kinda hidden down in the "Command Line" document.  Few people venture in there 😝

**Reply by meomeo0308** [reply]
If i use Docker Compose, what do i need to do to backup everything except job logs and job history? Thanks!

**Reply by meomeo0308** [reply]
Because when I use the command /opt/cronicle/bin/control.sh export and import, the container stops immediately and cannot be restarted.

**Reply by jhuckaby** [reply]
Well, Docker is designed to run a single command and then exit as soon as the command completes.  You don't want that in this case.  Sounds like you need to run your container with the "bash" command, and the `-it` param, so you have an interactive shell to run the command and get the file.  Also add a bind mount (`-v`) so you can grab files from outside the container.
